Source/WebKit/UIProcess/API/glib/WebKitInstallMissingMediaPluginsPermissionRequest.cpp

```diff
@@ -51,14 +41,13 @@ using namespace WebCore;
  * it is allowed by default.
  *
  * Since: 2.10
+ *
+ * Deprecated: 2.40
  */
 
 static void webkit_permission_request_interface_init(WebKitPermissionRequestIface*);
 
 struct _WebKitInstallMissingMediaPluginsPermissionRequestPrivate {
-#if ENABLE(VIDEO) && !USE(GSTREAMER_FULL)
-    RefPtr<InstallMissingMediaPluginsPermissionRequest> request;
-#endif
     CString description;
     bool madeDecision;
```

Comment:
I don't think these are needed anymore either, so you should be able to remove them.